apps/user_ldap/lib/Group_LDAP.php

```diff
@@ -786,6 +798,10 @@ public function getUserGroups($uid) {
 
 		$groups = array_unique($groups, SORT_LOCALE_STRING);
 		$this->access->connection->writeToCache($cacheKey, $groups);
+		/** @var IConfig $config */
+		$config = \OCP\Server::get(IConfig::class);
+		$groupStr = \json_encode($groups);
+		$config->setUserValue($ncUid, 'user_ldap', 'group-memberships-' . $this->access->connection->getConfigPrefix(), $groupStr);
```

Comment:
```suggestion
		$config->setUserValue($ncUid, 'user_ldap', 'cache-group-memberships-' . $this->access->connection->getConfigPrefix(), $groupStr);
```
I’d prefer something prefixed with cache or backup to avoid misusages of this configuration key when people will stumble upon it.
Also, for modern releases if we could avoid this config key and use the group membership mapper I’d feel better, but we need to check first if the group membership mapping is kept in these situations.